vmware-tanzu/tanzu-framework
# Migrate capabilities component to management package repo

### What this PR does / why we need it

This PR migrates capabilities component to management package repo so that it can lifecycle managed using Carvel tools

### Which issue(s) this PR fixes

Fixes # https://github.com/vmware-tanzu/tanzu-framework/issues/1234

### Describe testing done for PR
1. Build the package
```
make package-bundles
```
2. Build package repo bundle
```
PACKAGE_REPOSITORY=management OCI_REGISTRY=${REGISTRY} make package-repo-bundle
```
3. Verify build dir for capabilities package bundle and package generation in the project root dir
```
build
├── package-bundles
│   └── management
│       ├── capabilities-v0.11.0-dev.tar.gz
│       ├── core-management-plugins-v0.11.0-dev.tar.gz
│       └── featuregates-v0.11.0-dev.tar.gz
└── package-repo-bundles
    └── management
        ├── packages
        │   ├── capabilities.tanzu.vmware.com
        │   │   ├── 0.11.0-dev.yml
        │   │   └── metadata.yml
        │   ├── core-management-plugins.tanzu.vmware.com
        │   │   ├── 0.11.0-dev.yml
        │   │   └── metadata.yml
        │   └── featuregates.tanzu.vmware.com
        │       ├── 0.11.0-dev.yml
        │       └── metadata.yml
        ├── tanzu-framework-management-repo-v0.11.0-dev.tar.gz
        └── tanzu-framework-management-repo-v0.11.0-dev.yaml
```


### Release note
```release-note
Migrated capabilities controller to management package repository as a Carvel package
```

### PR Checklist



      


### Additional information

#### Special notes for your reviewer
